instrumentation/spring/spring-batch-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/batch/SpringBatchInstrumentationConfig.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.batch;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.unmodifiableList;
+
+import io.opentelemetry.instrumentation.api.config.Config;
+import java.util.List;
+
+public final class SpringBatchInstrumentationConfig {
+  private static final List<String> INSTRUMENTATION_NAMES =
+      unmodifiableList(asList("spring-batch", "spring-batch-3.0"));
+
+  public static List<String> instrumentationNames() {
+    return INSTRUMENTATION_NAMES;
+  }
+
+  public static boolean isTracingEnabled(String type) {
+    return Config.get()
+        .getInstrumentationBooleanProperty(instrumentationNames(), type + ".enabled", true);
```

Comment:
Looking at #1874, I think this should be disabled by default.

Reply:
Hm, I think that right now the whole instrumentations should be treated as experimental and turned off by default.
(And regarding the `isTracingEnabled()` method: honestly I'm not sure that they're needed at all, there's a good chance that I'll remove them in some later PR.)